_layouts/home.html

```diff
@@ -13,12 +13,99 @@ <h2 id="countdown">&nbsp;</h2>
     </p>
   </div>
 
-  <div class="flex row-xs-center ">
-    <div class="btn-group-lg">
-      <a class="btn btn-lg btn-default btn-next btn-xs-full" href="{{ '/host/' | prepend: site.baseurl }}">Become a host&nbsp;&nbsp;<i class="fa fa-play" aria-hidden="true"></i></a>
-      <a class="btn btn-lg btn-default btn-next btn-xs-full" href="{{ '/participant/' | prepend: site.baseurl }}">Attend an event&nbsp;&nbsp;<i class="fa fa-play" aria-hidden="true"></i></a>
-    </div>
-  </div>
+  <!-- Add this css File in head tag-->
+          <link href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css" rel="stylesheet" media="all">
+          <link href="https://cdnjs.cloudflare.com/ajax/libs/animate.css/3.5.2/animate.min.css" rel="stylesheet" media="all">
+
+          <div id="bootstrap-touch-slider" class="carousel bs-slider fade  control-round indicators-line" data-ride="carousel" data-pause="hover" data-interval="5000" >
```

Comment:
please note the correct indentation of the file. We are using 2 spaces here and the whole block should be further left to match the other lines.